### ISSUE TITLE
Doesn't work with Firefox 77.0.1

### ISSUE THREAD

**Issue by MasterGroosha** [StylishThemes/GitHub-Dark-Script#80]
I have two main devices: my PC and my laptop. Both have Windows installed.
On my PC, I remember Github dark broken a while ago, but on my laptop the script got broken after updating Firefox to 77.0.1.
I tried re-installing the script and even re-installing Tampermonkey, didn't help.

* **Browser**:  Firefox 77.0.1
* **Operating System**: Windows 10
* **Link to page with the issue**: Any github.com page
* **Screenshot**:
![изображение](https://user-images.githubusercontent.com/2780044/83930451-56239580-a7a0-11ea-83ed-0dc708a94504.png)

**Comment by precla** [comment]
Same here.
OS: Manjaro

**Comment by the-j0k3r** [comment]
Consider using Stylus and GitHub Dark UserCSS https://github.com/StylishThemes/GitHub-Dark#installation

You will have a better OOBE that way its more customizable and less buggy and includes multi @-moz-doc rules support for other GH sites, something the script cant.

**Comment by precla** [comment]
Thanks, works with Stylus.

**Comment by Newcool1230** [comment]
> 
> 
> Consider using Stylus and GitHub Dark UserCSS https://github.com/StylishThemes/GitHub-Dark#installation
> 
> You will have a better OOBE that way its more customizable and less buggy and includes multi @-moz-doc rules support for other GH sites, something the script cant.

I mean if you are going to say you support other methods of using darkmode, then saying "use another program and it will fix the issue" is not a solution.

```
Installation

    First, you need a userscript manager. Options include:
        Violentmonkey for Firefox
        Violentmonkey for Chrome
        Greasemonkey for Firefox
        Tampermonkey for Chrome
        other Browsers

```

**Comment by silverwind** [comment]
Works fine for me in Firefox 78 with Violentmonkey. No idea what could be causing it except a breaking change in Tampermonkey. Do you see any errors from the script in either the devtools (F12) or browser console (CTRL-Shift-J)?

**Comment by the-j0k3r** [comment]
>I mean if you are going to say you support other methods of using darkmode, then saying "use another program and it will fix the issue" is not a solution.

If you prefer not to switch or even try that's your choice. =)

Read https://github.com/StylishThemes/GitHub-Dark-Script/issues/73#issuecomment-553383622 and GHD has also dropped grunt to mention a few which the script never caught up with.

I'm merely suggesting that you would have a much better user experience with stylus and GitHub Dark usercss **This is a fact.- its updated almost daily.**. With Script no matter what you still see flashes of white when loading pages even when this script was working very well.

@silverwind I think the script repo should be archived, it was made to support what usercss now supports and usercss supports this much better, besides no one is really maintaining it and its served its purpose,

**Comment by silverwind** [comment]
Yeah I think I tend do agree with archiving. The UserCSS version is superior in every way (same configurability and instant style loading). I don't really understand why the Userscript version is getting considerable usage still.
